frameworks/PHP/hyperf/app/Controller/IndexController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * This file is part of Hyperf.
+ *
+ * @link     https://www.hyperf.io
+ * @document https://doc.hyperf.io
+ * @contact  group@hyperf.io
+ * @license  https://github.com/hyperf-cloud/hyperf/blob/master/LICENSE
+ */
+
+namespace App\Controller;
+
+use App\Model\Fortune;
+use App\Model\World;
+use Hyperf\HttpMessage\Stream\SwooleStream;
+use Hyperf\HttpServer\Annotation\Controller;
+use Hyperf\HttpServer\Annotation\GetMapping;
+use Hyperf\HttpServer\Contract\RequestInterface;
+use Hyperf\HttpServer\Contract\ResponseInterface;
+use Hyperf\Utils\Context;
+
+/**
+ * @Controller
+ */
+class IndexController
+{
+    /**
+     * @GetMapping(path="/json")
+     */
+    public function json()
+    {
+        return ['message' => 'Hello, World!'];
+    }
+
+    /**
+     * @GetMapping(path="/db")
+     */
+    public function db()
+    {
+        return World::find(random_int(1, 10000));
+    }
+
+    /**
+     * @GetMapping(path="/queries/[{queries}]")
+     */
+    public function queries($queries = 1, ResponseInterface $response)
+    {
+        $queries = $this->clamp($queries);
+
+        $rows = [];
+        while ($queries--) {
+            $rows[] = World::find(random_int(1, 10000))->toArray();
+        }
+
+        return $response->json($rows);
+    }
+
+    /**
+     * @GetMapping(path="/fortunes")
+     */
+    public function fortunes(ResponseInterface $response)
+    {
+        $rows = Fortune::all();
+
+        $insert = new Fortune();
+        $insert->id = 0;
+        $insert->message = 'Additional fortune added at request time.';
+
+        $rows->add($insert);
+        $rows = $rows->sortBy('message');
+
+        $html = '<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>';
```

Comment:
Please try to use a template engine in the fortunes test.
This isn't a realistic approach in a full stack framework.

Also the requeriments say:

- The sorted list must be provided to a server-side template and rendered to simple HTML (see below for minimum template). The resulting HTML table displays each Fortune's id number and message text.
- The HTML generated by the template must be sent as a response.
- The implementation is encouraged to use best practices for templates such as layout inheritence, separate header and footer files, and so on. However, this is not required. We request that implementations do not manage assets (JavaScript, CSS, images). We are deferring asset management until we can craft a more suitable test.